### PR TITLE
BUGZ-774: Lilypad Launcher (Mac) text issue

### PR DESCRIPTION
**Pull request by danteruiz**
Ticket - https://highfidelity.atlassian.net/browse/BUGZ-774

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20113/HighFidelity-Beta-PR15831-e119afd.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20113/HighFidelity-Beta-Interface-PR15831-e119afd.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20113/HighFidelity-Beta-PR15831-e119afd.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20113/HighFidelity-Beta-Interface-PR15831-e119afd.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20113/HighFidelity-Beta-PR15831-e119afd.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20113/HighFidelity-Quest-Beta-PR15831-e119afd.apk)

File: launchers/darwin/nib/DisplayNameScreen.xib [hunk 1]
```diff
@@ -69,8 +69,17 @@
                         <action selector="hyperLink:" target="YVh-OH-vU8" id="JfS-ot-wAP"/>
                     </connections>
                 </textField>
+                <textField horizontalHuggingPriority="251" verticalHuggingPriority="750" fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="0CN-kS-xZ5">
+                    <rect key="frame" x="66" y="68" width="266" height="17"/>
+                    <autoresizingMask key="autoresizingMask"/>
+                    <textFieldCell key="cell" lineBreakMode="clipping" alignment="center" title="By Signing in, you agree to the High Fidelity" id="zSf-YA-osu">
```
Comment (huffman) [inline launchers/darwin/nib/DisplayNameScreen.xib]: I believe that "Signing" here should be lowercase

Comment (danteruiz) [reply launchers/darwin/nib/DisplayNameScreen.xib]: right.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20117/HighFidelity-Beta-PR15831-dd639c4.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20117/HighFidelity-Beta-Interface-PR15831-dd639c4.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20117/HighFidelity-Beta-PR15831-dd639c4.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20117/HighFidelity-Beta-Interface-PR15831-dd639c4.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20117/HighFidelity-Beta-PR15831-dd639c4.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20117/HighFidelity-Quest-Beta-PR15831-dd639c4.apk)

**Pull request merged**